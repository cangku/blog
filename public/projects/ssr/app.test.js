import React from 'react';
import App from './app';
import renderer from 'react-test-renderer';

// https://reactjs.org/docs/test-renderer.html
test('react component', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
    expect(tree.children[0]).toBe('app component');
    expect(tree.children[1].children[0]).toBe('3');
})