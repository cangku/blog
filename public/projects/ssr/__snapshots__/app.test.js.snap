// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react component 1`] = `
<div>
  app component
  <span>
    3
  </span>
</div>
`;
